src/Sulu/Bundle/ContentBundle/Content/Structure/ExcerptStructureExtension.php

```diff
@@ -227,4 +254,79 @@ private function initProperties($locale)
             $this->properties[] = $property->getName();
         }
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function export($properties, $format = null)
+    {
+        $container = new ExcerptValueContainer($properties);
+
+        $data = [];
+        foreach ($this->getExcerptStructure()->getProperties() as $property) {
+            if ($container->__isset($property->getName())) {
+                $property->setValue($container->__get($property->getName()));
+                $contentType = $this->contentTypeManager->get($property->getContentTypeName());
+                if ($this->contentExportManager->hasExport($property->getContentTypeName(), $format)) {
+                    $options = $this->contentExportManager->getOptions($property->getContentTypeName(), $format);
+
+                    $data[$property->getName()] = [
+                        'name' => $property->getName(),
+                        'value' => $contentType->exportData($property->getValue()),
+                        'type' => $property->getContentTypeName(),
+                        'options' => $options,
+                    ];
+                }
+            }
+        }
+
+        return $data;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getImportPropertyNames()
+    {
+        $propertyNames = [];
+
+        foreach ($this->getExcerptStructure()->getProperties() as $property) {
+            $propertyNames[] = $property->getName();
+        }
+
+        return $propertyNames;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function import(NodeInterface $node, $data, $webspaceKey, $languageCode, $format)
+    {
+        $this->setLanguageCode($languageCode, 'i18n', null);
+
+        foreach ($this->getExcerptStructure()->getProperties() as $property) {
+            $contentType = $this->contentTypeManager->get($property->getContentTypeName());
+
+            if (
```

Comment:
then its also to long with the line break because you have to indent the next line with 4 spaces which is exactly the the same size as `if (`